tensornetwork/backends/backend_test.py

```diff
@@ -0,0 +1,58 @@
+"""Tests for graphmode_tensornetwork."""
+import builtins
+import sys
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def clean_backend_import():
+  #never do this outside testing
+  sys.modules.pop('tensornetwork.backends.pytorch.pytorch_backend', None)
+  sys.modules.pop('tensornetwork.backends.jax.jax_backend', None)
+  sys.modules.pop('tensornetwork.backends.tensorflow.tensorflow_backend', None)
+  sys.modules.pop('tensornetwork', None)
+  yield # use as teardown
+  sys.modules.pop('tensornetwork', None)
+
+
+@pytest.fixture
+def no_backend_dependency(monkeypatch):
+  import_orig = builtins.__import__
+  # pylint: disable=redefined-builtin
+  def mocked_import(name, globals, locals, fromlist, level):
+    if name in ['torch', 'tensorflow', 'jax']:
+      raise ImportError()
+    return import_orig(name, globals, locals, fromlist, level)
+  monkeypatch.setattr(builtins, '__import__', mocked_import)
+
+
+@pytest.mark.usefixtures('no_backend_dependency')
+def test_backend_pytorch_missing():
+  with pytest.raises(ImportError):
+    from tensornetwork.backends.pytorch.pytorch_backend import PyTorchBackend
+
+
+@pytest.mark.usefixtures('no_backend_dependency')
+def test_backend_tensorflow_missing():
+  with pytest.raises(ImportError):
+    import tensornetwork.backends.tensorflow.tensorflow_backend
+
+
+@pytest.mark.usefixtures('no_backend_dependency')
+def test_backend_jax_missing():
+  with pytest.raises(ImportError):
+    from tensornetwork.backends.jax.jax_backend import JaxBackend
+
+
+@pytest.mark.usefixtures('no_backend_dependency')
+def test_config_pytorch_missing():
+  import tensornetwork.config
+  with pytest.raises(ImportError):
+    import torch
+
+
+@pytest.mark.usefixtures('no_backend_dependency')
```

Comment:
Can you add a test where we just import `tensornetwork` with all of the backends removed and make sure that doesn't raise an error? That should cover 90% of the mistakes we make again in the future.